position/errors.gno

```diff
@@ -0,0 +1,24 @@
+package position
+
+import (
+	"errors"
+
+	"gno.land/p/demo/ufmt"
+)
+
+var (
+	errNoPermission  = errors.New("[GNOSWAP-POSITION-001] no permission")
+	errSlippage      = errors.New("[GNOSWAP-POSITION-002] slippage")
+	errWrapUnwrap    = errors.New("[GNOSWAP-POSITION-003] wrap/unwrap")
+	errOutOfRange    = errors.New("[GNOSWAP-POSITION-004] out of range")
+	errInvalidInput  = errors.New("[GNOSWAP-POSITION-005] invalid input")
+	errDataNotFound  = errors.New("[GNOSWAP-POSITION-006] data not found")
+	errExpired       = errors.New("[GNOSWAP-POSITION-007] expired")
+	errWugnotMinimum = errors.New("[GNOSWAP-POSITION-008] wugnot minimum")
```

Comment:
What does this error mean?

Reply:
same as https://github.com/gnoswap-labs/gnoswap/pull/355#discussion_r1811787519